setup_default_backend.sh

```diff
@@ -157,19 +168,16 @@ fi
 
 DEFAULTDIR=$(pwd)
 export DEFAULTDIR
-echo "DEFAULTDIR of tracim => \"$DEFAULTDIR\""
+echo "Tracim DEFAULTDIR: $DEFAULTDIR"
 
 install_npm_and_nodejs
-log "go to backend subdir.."
-cd $DEFAULTDIR/backend  || exit 1
-install_backend_system_dep
+cd "$DEFAULTDIR/backend"  || exit 1
+if [ -z "$IGNORE_APT_INSTALL" ]; then
+    install_backend_system_dep
+fi
 setup_pyenv
 install_backend_python_packages
 setup_config_file
 create_require_dirs
 setup_db
 translate_email
-
-# Return to "$DEFAULTDIR/"
-log "cd $DEFAULTDIR"
-cd $DEFAULTDIR || exit 1
```

Comment:
Does this mean the new script change current dir when script end ?

Reply:
I removed this because it has no effect.